decred/decred/dcr/rpc.py

```diff
@@ -3286,3 +3366,85 @@ def parse(obj):
             prerelease=obj["prerelease"],
             buildMetadata=obj["buildmetadata"],
         )
+
+
+def websocketURI(host):
+    """
+    Parse the dcrd websocket URI from the HTTP URI.
+
+    Args:
+        host (str): The host.
+
+    Returns:
+        str: The websocket URI.
+    """
+    uri = urlparse(host)
+    prot = "wss" if uri.scheme in ("https", "wss") else "ws"
+    return f"{prot}://{uri.netloc}/ws"
```

Comment:
We already use functions from `urllib.parse` in several places, I wonder if we want to use them more systematically.

Reply:
Probably. Feel free to follow up if you've got something in mind.